package.json

```diff
@@ -61,7 +61,7 @@
     "npm-run-all": "4.1.5",
     "package-json-reducer": "1.0.18",
     "prettier": "~2.3.0",
-    "semantic-release": "^23.0.7",
+    "semantic-release": "24.2.3",
```

Comment:
Might as well live on the bleeding edge :joy:

Reply:
Is there a reason you pinned the dependency, rather than using `^`?

Reply:
Not a good enough reason so that I could defend it :joy:   I'll add it back.

Reply:
updated it